### ISSUE TITLE
[Docs][Bug] copy custom selection of code removes line breaks in Firefox

### ISSUE THREAD

**Issue by auipga** [zmkfirmware/zmk#1014]
**Steps to reproduce:**
- go to https://zmk.dev/docs/behaviors/mod-morph (as on example)
- copy multiple lines with <kbd>Ctrl</kbd>+<kbd>C</kbd> or context menu:
  ![image](https://user-images.githubusercontent.com/876689/140078712-4241b095-b404-4121-80d4-a933c4a6b10b.png)

**Expected result on paste:**
```
        gresc: grave_escape {
            compatible = "zmk,behavior-mod-morph";
            label = "GRAVE_ESCAPE";
            #binding-cells = <0>;
            bindings = <&kp ESC>, <&kp GRAVE>;
            mods = <(MOD_LGUI|MOD_LSFT|MOD_RGUI|MOD_RSFT)>;
        };
```

**Actual result on paste:** 
```
        gresc: grave_escape {            compatible = "zmk,behavior-mod-morph";            label = "GRAVE_ESCAPE";            #binding-cells = <0>;            bindings = <&kp ESC>, <&kp GRAVE>;            mods = <(MOD_LGUI|MOD_LSFT|MOD_RGUI|MOD_RSFT)>;        };
```
---
Details: Firefox 94.0, Linux (Archlinux)
Cross-check with Vivaldi (based on Chromium): bug not present there.

**Comment by joelspadin** [comment]
We need to pull in this fix: https://github.com/facebook/docusaurus/issues/5663

**Comment by caksoylar** [comment]
This seems to be fixed now as I can't reproduce on Firefox 119.0 on Windows, possibly by the Docusaurus update in #1779. Please re-open if it is still an issue.
